nservicebus/messaging/data-distribution.md

```diff
@@ -1,40 +1,40 @@
+A common example of a data distribution scenario is having cached data on multiple, scaled-out web servers, and attempting to deliver a message to each of them which indicates that they should drop their current cache entries and retrieve fresh data from the database.
 
 ## Timing and reliability
 
-The goal of data distribution is generally to cause a change in data and/or behavior in multiple locations at the same time. Messages sent with NServiceBus, however do not care if they are received or processed at the same time. There is no urgency to act _now_ with a message sent with NServiceBus. Because asynchronous messages are intended for eventual consistency, each receiver is free to process it at their leisure, whenever it gets around to handling that message in the queue.
+The goal of data distribution is generally to cause a change in data and/or behavior in multiple locations at the same time. NServiceBus does not provide any facility for receiving or processing messages in multiple locations at the same time. There is no urgency to act _now_ in reaction to a message sent with NServiceBus. Asynchronous messages are intended for eventual consistency. Each receiver processes a given message whenever it happens to receive that message from the queue.
 
-In the worst-case scenario, some receivers of a data distribution message might succeed, while others encounter an unknown error causing the message to be moved to the error queue. In the time until an administrator is able to diagnose the failure and return the message to its source queue, transactions would be handled very differently by nodes that had received the message and those that had not.
+In the worst-case scenario, some receivers of a data distribution message may succeed in handling the message, and others may fail, causing the message to be moved to the error queue. Until an administrator diagnoses the failure and returns the message to its source queue, transactions may be handled very differently by nodes that had successfully handled the message and those that had not.
 
-These sorts of failures are difficult to track down, as they depend upon more than the contents of a message and the code processing it. The result would also be dependent upon the state of the data being handled by data distribution, which would not be consistent between different servers in a cluster.
+These kinds of failures are difficult to track down. They depend upon more than the contents of a message and the code handling it. The results are also dependent upon the state of the data being handled by data distribution, which will not be consistent across all servers in a cluster.
 
-## Differences from publish/subscribe
+## Differences to publish/subscribe
```

Comment:
I can't back this up, but "difference to" / "different to" sounds unnatural to my ear. (Next paragraph and elsewhere too.) Is this American English vs. English English at play?

Reply:
> Is this American English vs. English English at play?

I think it [could be](https://english.stackexchange.com/a/845). I'll revert it to _from_.